code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/github/GithubIdentitySearchProvider.java

```diff
@@ -81,13 +82,18 @@
 
     @Override
     public Set<Identity> getIdentities(Account account) {
-        if (!isConfigured() && !GithubConstants.CONFIG.equalsIgnoreCase(SecurityConstants.AUTH_PROVIDER.get())) {
+        if (!isConfigured() || !GithubConstants.CONFIG.equalsIgnoreCase(SecurityConstants.AUTH_PROVIDER.get()) ||
+                !GithubConstants.USER_SCOPE.equalsIgnoreCase(account.getExternalIdType())) {
             return new HashSet<>();
         }
-        githubUtils.findAndSetJWT();
```

Comment:
Can you provide some background or explain the changes in this file?  Not questioning your changes.  Just need some background to understand.  Thanks!